packages/document/docs/en/shared/config/server/publicDir.md

```diff
@@ -0,0 +1,19 @@
+- **Type:** `string | false`
```

Comment:
How about adding `copyOnBuild` as a property of this option?

```ts
export default {
  server: {
    publicDir: {
      name: 'public',
      copyOnBuild: true,
    }
  }
}
```

The object structure allows us to add more options for publicDir in the future.

Reply:
Already supported